homeassistant/components/daikin_madoka/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""Platform for the Daikin AC."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from pymadoka import Controller, discover_devices, force_device_disconnect
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_DEVICE,
+    CONF_DEVICES,
+    CONF_FORCE_UPDATE,
+    CONF_SCAN_INTERVAL,
+)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
```

Comment:
Not sure if this is still the best way, or if we just use `HomeAssistant` from `from homeassistant.core` nowadays.

Reply:
Updated